tools/ALARAJOYWrapper/tendl_processing.py

```diff
@@ -192,35 +245,53 @@ def iterate_MTs(MTs, file_obj, mt_dict, pKZA, all_rxns):
                     }
                 }    
             }
+        radionucs (dict): Dictionary keyed by all radionuclides in the EAF
+            decay library, with values of their half-lives.
             
     Returns:
         all_rxns (collections.defaultdict): Updated dictionary for all
             reaction pathways for the given parent and its MTs.
     """
 
     for MT in MTs:
+        rxn = mt_dict[MT]
         sigmas = extract_cross_sections(file_obj, MT)
-        gas = mt_dict[MT]['gas']
+        gas = rxn['gas']
+
         # Daughter calculated either as an emitted gas nucleus or
-        # as the residual for non-gaseous emissions. 
+        # as the residual for non-gaseous emissions.
         dKZA = (
             GAS_DF.loc[GAS_DF['gas'] == gas, 'kza'].iat[0] if gas
-            else pKZA + mt_dict[MT]['delKZA']
+            else pKZA + rxn['delKZA']
         )
 
-        # Skip high (2 digit) isomeric states
-        if not mt_dict[MT]['high_m']:
+        # Process all reactions producing isomer daughters with decay data
+        # or any ground-state daughters. Necessarily need to cut off maximum
+        # excitation at 9th state by nature of KZA conventions
+        if (
+            _is_ground_state_nuc(rxn['isomer']) or
+            _is_isomer_with_decay_data(dKZA, radionucs, rxn['isomer'])
+        ):
             all_rxns[pKZA][dKZA][MT] = {
-                'emitted'               :              mt_dict[MT]['emitted'],
-                'xsections'             :                             np.pad(
+                'emitted'    :  rxn['emitted'],
+                'xsections'  :  np.pad(
                     sigmas, (0, VITAMIN_J_ENERGY_GROUPS - len(sigmas))
                 )
             }
 
+        # If an (n,n) reaction produces an isomer lacking decay data,
+        # accumulate its cross-sections to the ground-state residual (n,n)
+        # reaction's cross-sections
         else:
-            warnings.warn(f'''
-                Skipping high isomeric state in daughter for {pKZA} → {dKZA}.
-                MT > 9 not allowed by ALARA.
-            ''')
+            nn_MT = -4 # Negative parallel to MT=4 for standard (n,n) reaction
+            if nn_MT not in all_rxns[pKZA][pKZA]:
+                all_rxns[pKZA][pKZA][nn_MT] = {
+                    'emitted'   : 'n',
```

Comment:
Calling this `n*` helps indicate that it's leaving the child in an excited state.  Not sure if this is consistent with other places in the MT metadata, though?
```suggestion
                    'emitted'   : 'n*',
```